pymemcache/client/hash.py

```diff
@@ -445,3 +445,46 @@ def flush_all(self, *args, **kwargs):
     def quit(self):
         for client in self.clients.values():
             self._safely_run_func(client, client.quit, False)
+
+    def raw_command(self, command, end_tokens="\r\n", send_to_all=False):
+        """
+        Sends an arbitrary command to the server and parses the response until a
+        specified token is encountered.
+
+        OBS: If the end_tokens are not included in the reply, the client will wait until it's
```

Comment:
I'm not sure what `OBS` means?

Reply:
Oh, that might be a Danish-ism I let slip in. Just short of "observe" could also be read as "NOTE" or "take not of".

Reply:
Ah, we often see "NB" ([Nota bene](https://en.wikipedia.org/wiki/Nota_bene)) in the US instead.

Let's avoid using a shorthand either way. :)